### ISSUE TITLE
Learning rate decay

### ISSUE THREAD

**Issue by djsaunde** [Hananel-Hazan/bindsnet#291]
We've no in-library implementation of learning rate decay. It could be implemented similarly to weight decay in `Connection` objects, but in `LearningRule` objects, operating on `nu`.

**Comment by tomking** [comment]
Hello, sorry if this is an erroneous question, I am just starting to learn about this library. 

Would it make sense for this feature to implement the option of defining individual learning rate decays to operate on pre and post synaptic events in addition to operating on both of them with the same value?

**Comment by Hananel-Hazan** [comment]
Yes, it may help, but for that we need to find the relevant biological evidence for this separate adjustment and also application that can utilize it.
